src/hooks/useCurrentSeason.ts

```diff
@@ -0,0 +1,27 @@
+import { DateTime } from 'luxon';
+
+const seasons = new Map<number, DateTime>();
+const SEASON_START_DATE = DateTime.fromISO('2022-08-01T16:03:44+00:00');
+const SEASON_DURATION = 1; // months
```

Comment:
the duration should be one quarter (on average 3 months)

Reply:
i'd probably just have a mapping with season start timestamp and season end timestamp that we can configure and set ourselves, wdyt?

Reply:
fixed in: 075cc4da9fa1be34e1cb2de394276557873761ca. Check useCurrentSeason/constants.ts